<?xml version="1.0"?>
<launch>
  <arg name="side" default="r"/> <!-- "b" = blue side, "r" = red side  -->

    <!-- You can change control program  -->

    <!-- sample program node -->
    <!-- <node pkg="burger_war" type="randomRun.py" name="randomRun" output="screen"/> -->
    <!-- END sample program node -->

    <!-- PARAMS FOR DEBBUG -->    
    
    <!-- make args to params -->
    <group if="$(eval side=='r')">
      <param name="side" value="r"/>          
    </group>
    <group if="$(eval side=='b')">
      <param name="side" value="b"/>          
    </group>
    
    <node pkg="burger_war" type="war_state_publisher.py" name="warState" output="screen"/>    
    
    <!-- End of your space  -->
</launch>

